src/components/Tag/tests/Tag.test.tsx

```diff
@@ -35,4 +37,85 @@ describe('<Tag />', () => {
       expect(spy).not.toHaveBeenCalled();
     });
   });
+
+  describe('url', () => {
+    it('renders an anchor tag when url is provided', () => {
+      const tag = mountWithApp(<Tag url="#" />);
+      expect(tag).toContainReactComponent('a', {href: '#'});
+    });
+
+    it('allows url and onRemove props at the same time', () => {
+      const spy = jest.fn();
+      const tag = mountWithApp(<Tag onRemove={spy} url="#" />);
+      tag.find('button')!.domNode!.click();
+      expect(spy).toHaveBeenCalled();
+      expect(tag).toContainReactComponent('a', {href: '#'});
+    });
+
+    it('renders plain text when the tag is disabled', () => {
```

Comment:
Should we also have a test that asserts a span is rendered instead of anchor when url and disabled are passed in?